log/severity_string.go

```diff
@@ -0,0 +1,47 @@
+// Code generated by "stringer -type=Severity -linecomment"; DO NOT EDIT.
+
+package log
+
+import "strconv"
+
+func _() {
+	// An "invalid array index" compiler error signifies that the constant values have changed.
+	// Re-run the stringer command to generate them again.
+	var x [1]struct{}
+	_ = x[SeverityTrace1-1]
+	_ = x[SeverityTrace2-2]
+	_ = x[SeverityTrace3-3]
+	_ = x[SeverityTrace4-4]
+	_ = x[SeverityDebug1-5]
+	_ = x[SeverityDebug2-6]
+	_ = x[SeverityDebug3-7]
+	_ = x[SeverityDebug4-8]
+	_ = x[SeverityInfo1-9]
+	_ = x[SeverityInfo2-10]
+	_ = x[SeverityInfo3-11]
+	_ = x[SeverityInfo4-12]
+	_ = x[SeverityWarn1-13]
+	_ = x[SeverityWarn2-14]
+	_ = x[SeverityWarn3-15]
+	_ = x[SeverityWarn4-16]
+	_ = x[SeverityError1-17]
+	_ = x[SeverityError2-18]
+	_ = x[SeverityError3-19]
+	_ = x[SeverityError4-20]
+	_ = x[SeverityFatal1-21]
+	_ = x[SeverityFatal2-22]
+	_ = x[SeverityFatal3-23]
+	_ = x[SeverityFatal4-24]
+}
+
+const _Severity_name = "TRACETRACE2TRACE3TRACE4DEBUGDEBUG2DEBUG3DEBUG4INFOINFO2INFO3INFO4WARNWARN2WARN3WARN4ERRORERROR2ERROR3ERROR4FATALFATAL2FATAL3FATAL4"
```

Comment:
I haven't seen this pattern of `_Foo` before.  Is this constant visible outside of the package?